app/models/supplier_category.rb

```diff
@@ -0,0 +1,19 @@
+class SupplierCategory < ActiveRecord::Base
+
+  belongs_to :financial_transaction_class
+  has_many :suppliers
```

Comment:
idem

Reply:
there is a custom `check_for_associated_suppliers` for it